frontend/components/main/feature.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import store from '../../store/store.js'
+import {getFeature} from '../../actions/feature-action.js'
+
+const Feature = React.createClass({
+	render(){
+		console.log('FEATURE ===>',this.props.featureProps)
+		// let feature = this.props.featureProps[0]
+		return(
+			<div>
+			Feature
+			Name: 
+			Quote: 
+			Image: 
+			{store.dispatch(getFeature(2))}
```

Comment:
i need to see what if my action was working, still working on this feature component